### DISCUSSION TITLE
How to achieve "Galaxy Brain" badge?

### DISCUSSION THREAD

**Discussion by mr-azharul** [Schweinepriester/github-profile-achievements#592] [Q&A] [answered]
There is a achievement badge in github name "Galaxy Brain", how to achieve that?

**Comment by Turza006** [comment, marked as answer]
The "Galaxy Brain" achievement badge on GitHub is awarded for making a large number of contributions (e.g. commits, pull requests) in a short period of time. The specific requirements for earning the badge may vary depending on the time period and repository, but some possible strategies for achieving it include:

- Collaborating on a popular open-source project with a high rate of contributions
- Participating in a GitHub hackathon or other event that encourages rapid contributions
- Making a large number of small commits or pull requests to a project over a short period of time

It's worth noting that this badge is awarded automatically by GitHub and there is no way to directly apply for it. The best way to achieve 
this is to make a lot of contributions to a project and it is given to the developer who has made a lot of contributions in a short amount of time.
